packages/web-components/@sfx/product/stories/index.ts

```diff
@@ -0,0 +1,103 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import '../src/index.ts';
+import Product, { ProductModel } from '../src/product';
+
+const product = <ProductModel> {
+  name: 'Cool Product',
+  price: 39.99,
+  label: 'New Product',
+  promo: '25% off',
+  imageSrc: 'https://spudsmart.com/wp-content/uploads/2017/10/NZ-Bag-of-potatoes-cropped.png',
+  imageAlt: 'Good bag of potatoes'
+};
+
+storiesOf('Components|Product', module)
+  .addDecorator(withKnobs)
+  .add('Default', () => `
+    <sfx-product 
+      product="${text('Product Info', JSON.stringify({
+        ...product,
+        variants: { 
+          type: 'color', 
+          items: [
+            { color: '#da5', text: 'Brown', product},
+            { color: '#7a0', text: 'Green', product: { 
+              imageSrc: 'https://teara.govt.nz/files/p-9780-enz.jpg' },
+              imageAlt: 'Spicy green potatoes'
+            }, 
+            { color: '#546', text: 'Blue', product: { 
+              imageSrc: 'http://startuptipsdaily.com/wp-content/uploads/2016/12/purple-and-blue-potato.jpg' },
+              imageAlt: 'Mystery blue potatoes'
+            }, 
+          ] 
+        }
+      }))}"
+    ></sfx-product>
+  `, { 
+    notes: { 
+      markdown: `
+      # SF-X Product Component
```

Comment:
I think everything in here should be indented.

Reply:
Wondering if it's worth it to write something that will pull in the package's README.md file if it exists and if it doesn't or there's a string here it will be overwritten.